testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/delta/BaseDeltaFailureRecoveryTest.java

```diff
@@ -86,14 +86,12 @@ public void testDelete()
                 .finishesSuccessfully();
 
         // DELETE plan is too simplistic for testing with `intermediateDistributedStage`
-        assertThatThrownBy(() ->
-                assertThatQuery(deleteQuery)
-                        .withSetupQuery(setupQuery)
-                        .withCleanupQuery(cleanupQuery)
-                        .experiencing(TASK_FAILURE, Optional.of(ErrorType.INTERNAL_ERROR))
-                        .at(intermediateDistributedStage())
-                        .failsWithoutRetries(failure -> failure.hasMessageContaining(FAILURE_INJECTION_MESSAGE)))
-                .hasMessageContaining("stage not found");
+        assertThatQuery(deleteQuery)
```

Comment:
If DELETE plan is now complex enough for failure injection we can remove the comment above too

Reply:
Removed.